src/components/Chat/InputMessageV2.tsx

```diff
@@ -158,7 +360,7 @@ const InputMessageV2 = ({
 
     Alert.alert(
       '',
-      type === 'ANONYMOUS'
+      type === ANONYMOUS
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Alert.alert` function has been updated to use constants for the chat types instead of hardcoded strings. This is a good change as it reduces the risk of typos and makes the code easier to maintain.



<!-- This is an auto-generated comment by OSS CodeRabbit -->